src/Formatter/IntlMoneyFormatter.php

```diff
@@ -16,13 +16,19 @@
      * @var \NumberFormatter
      */
     private $formatter;
+    /**
+     * @var int|null
```

Comment:
@sagikazarmark The case: "we are not able to control fraction digits anymore, but only subunits".

Look at the last two commits I added, specifically `testNoFractionDigitsNoSubunitsNoPattern`. The default behaviour that `intl` gives us is fine for the majority of the cases. The case that @bramstroker has added is the case when you want to display a different amount of fractions than the number of subunits that a currency has.

That is what this PR fixes. You now have control over both.